### PR TITLE
Feature/add fisheye model support

### PR DESCRIPTION
**Pull request by PerrineAguiar**
Adds support for fisheye model in `image_geometry`.

### CODE REVIEW

File: image_geometry/include/image_geometry/pinhole_camera_model.h [hunk 1]
```diff
@@ -23,6 +23,7 @@ class Exception : public std::runtime_error
 class PinholeCameraModel
 {
 public:
+  static const std::string DISTORTION_MODEL_FISHEYE;
```
Comment (meyerj) [inline image_geometry/include/image_geometry/pinhole_camera_model.h]: nit: Is this constant needed by users of the class? If not, you could declare it as a static global in [pinhole_camera_model.cpp](https://github.com/Intermodalics/vision_opencv/pull/1/files#diff-dd4b759a7ec4bef379b85ef5ce2a5a23) and not modify the header at all. Publishers of camera_info typically do not include the header and need to know the string to assign to `distortion_model` from somewhere else anyway.

**Review by meyerj — APPROVED** [review]
Tested and released as a Debian to override upstream `ros-kinetic-image-geometry`. The release branches have been pushed directly to this repo.

In order to update the Debian:
1. Update the `upstream` branch to whatever branch you want to release.
2. Run `git-bloom-release kinetic`.
3. Checkout branch [debian/ros-kinetic-image-geometry_x.y.z-i_xenial](https://github.com/Intermodalics/vision_opencv/tree/debian/ros-kinetic-image-geometry_1.12.8-0_xenial).
4. Run `dpkg-buildpackage -us -uc -b`.
5. Push all branches and tags.

**Comment by PerrineAguiar** [comment]
@meyerj Nit addressed.
And I added support for the opencv fisheye model.

**Comment by meyerj** [comment]
New debian built and uploaded to http://packages.intermodalics.eu/ros/ubuntu/pool/main/r/ros-kinetic-image-geometry/ros-kinetic-image-geometry_1.12.8-2xenial-intermodalics_amd64.deb.

LGTM!

**Comment by meyerj** [comment]
The `EQUIDISTANT` model that is equivalent to our `FISHEYE_CV` has been added in https://github.com/ros-perception/vision_opencv/pull/358.

**Pull request merged**